test/unit/classification_test.rb

```diff
@@ -410,7 +410,7 @@ def setup
 
   test 'smart variable of hash with merge_overrides and priority should return lookup_value hash with array of elements' do
     key = FactoryGirl.create(:lookup_key, :key_type => 'hash', :merge_overrides => true,
-                             :default_value => {}, :path => "organization\nos\nlocation",
+                             :default_value => {:default => 'example'}, :path => "organization\nos\nlocation",
```

Comment:
To show that it doesn't change merge_override's behavior when it's set to false, I'll divide into 2 tests.